template-parts/content/content-page.php

```diff
@@ -14,8 +14,13 @@
 <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
 
 	<header class="entry-header alignwide">
-		<?php get_template_part( 'template-parts/header/entry-header' ); ?>
-		<?php twenty_twenty_one_post_thumbnail(); ?>
+		<?php
+		if ( ! is_front_page() ) {
+			get_template_part( 'template-parts/header/entry-header' );
+		}
+
+			twenty_twenty_one_post_thumbnail();
+		?>
```

Comment:
Templates should use the alternative syntax for conditions (see suggestion below)

```suggestion
		<?php if ( ! is_front_page() ) : ?>
			<?php get_template_part( 'template-parts/header/entry-header' ); ?>
		<?php endif; ?>
		<?php twenty_twenty_one_post_thumbnail(); ?>
```

Reply:
@aristath Thankyou for pointing this out, we were not aware of this. Is there some documentation on this subject?

Reply:
Unfortunately no... It's not a hard requirement just a convention & best-practice.
Template files should keep the logic to a minimum, and contain mostly HTML. In case logic is contained (like for example conditions like this one) it's traditional to not do it in PHP blocks but instead using a more HTML-like formatting so that people with no PHP experience don't get lost. That way it's easier for them to copy-paste things in a child theme and tweak things with minimal effort. If on the other hand PHP opening & closing tags are on separate lines, someone who doesn't know PHP can easily skip the closing tag and wonder why their site fails. Keeping things as self-contained as humanly possible makes it simple for non-developers to work on some things.